Dockerfile

```diff
@@ -1,30 +1,54 @@
-FROM node:14.16.1-slim
-
-RUN apt-get update && \
-  apt-get upgrade -yq && \
-  apt-get install -yq yarn git zlib1g zlib1g-dev
-
+FROM node:14-alpine as base
 WORKDIR /app
+RUN apk add --no-cache bash
 
-COPY entrypoint.sh .
-
-WORKDIR /app/build
-
+# get production dependencies
+FROM base as dependencies
 COPY . .
+RUN yarn install --prod
 
-# The following are all collapsed to reduce image size
-RUN yarn install &&\
-  yarn bazel clean &&\
-  yarn bazel build //comms/lighthouse:server &&\
-  yarn bazel build //content:server &&\
-  yarn bazel build //lambdas:server &&\
-  cp -L -R dist/bin/ ../bin &&\
-  yarn bazel clean --expunge && yarn cache clean &&\
-  cd .. &&\
-  rm -rf build &&\
-  rm -rf /root/.cache/bazel
-
-WORKDIR /app
+# build sources
+FROM base as catalyst-builder
+COPY . .
+RUN yarn install
+
+RUN yarn workspace @catalyst/contracts build
+RUN yarn workspace @catalyst/commons build
```

Comment:
What are those two doing? Was that already done? I'm not sure if we are using the contracts folder

Reply:
`commons` package depends on `contracts`, but not sure if we need to build them explicitly, will check if we can avoid doing this.